### DISCUSSION TITLE
No HiFi without Shairport Sync

### DISCUSSION THREAD

**Discussion by MSL-DA** [mikebrady/shairport-sync#1701] [General]
I have used Shairport Sync for several years and my setup would never be as good without it. I just want to say thank you for your work mikebrady ... I know there are PnP devices out there but none that offer the same flexibility as we get with Shairport Sync and your willingness to accommodate requests and solve problems is amazing and on a friendly manner. Thanks 🙏

**Comment by mikebrady** [comment]
Many thanks for your kind words! 😊
